models/logs_gather_cmd_request.go

```diff
@@ -0,0 +1,215 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
```

Comment:
Try rerunning without changing the swagger and ensure the validations aren't changed.